databuilder/models/dashboard/dashboard_owner.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+from typing import Optional, Dict, Any, Union, Iterator  # noqa: F401
+
+from databuilder.models.dashboard_metadata import DashboardMetadata
+from databuilder.models.neo4j_csv_serde import (
+    Neo4jCsvSerializable, RELATION_START_KEY, RELATION_END_KEY, RELATION_START_LABEL,
+    RELATION_END_LABEL, RELATION_TYPE, RELATION_REVERSE_TYPE)
+from databuilder.models.owner_constants import OWNER_OF_OBJECT_RELATION_TYPE, OWNER_RELATION_TYPE
+from databuilder.models.user import User
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class DashboardOwner(Neo4jCsvSerializable):
+    """
+    A model that encapsulate Dashboard's owner.
+    Note that it does not create new user as it has insufficient information about user but it builds relation
```

Comment:
what is a typical user for dashboard? Given we use email as user node key, should we still reuse user node for owner and figure out which user is the owner based on the email?

The concern I have will be that we would like to show the dashboard that people owns as well, folllow, user. I am not sure how it is easy to do if we add another different node for dashboard resource.